.travis.yml

```diff
@@ -1,26 +1,45 @@
-sudo: enabled
-
+dist: xenial
+sudo: required
 language: c
 
 install:
-    - make build-depends
-    - sudo apt-get -y install debian-archive-keyring dpkg=1.17.5ubuntu5.8
+  - make build-depends
+  - sudo apt-get -y install debian-archive-keyring
 
 # See the Ubuntu section in the debian/README file for more details, but this
 # is a good example of why CI is useful /before/ shipping a product..
 #
 # Also, the age of the debian-archive-keyring (2012) is simply concerning. So,
 # I have to manually add the debian stable release keys
 before_script:
-    - sudo sed -i '989s/$forceyes//' /usr/sbin/multistrap
-    - sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 7638D0442B90D010
-    - sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys EF0F382A1A7B6500
+  - sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 7638D0442B90D010
+  - sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys EF0F382A1A7B6500
 
 script:
-    - make test CONFIG_DISABLE_KVM=yes
-    - sha1sum output/*
-    - ls -al output/*
+  - make test CONFIG_DISABLE_KVM=yes
+  - sha1sum output/*
+  - ls -al output/*
+
+after_success:
+  - rename "s/.img/-$TRAVIS_TAG.img/" output/*.img
```

Comment:
I disagree with adding the tag to the image name.  The tag is contained in the release object created in github and the individual files do not need the tag - it makes it harder to have stable names for things and harder to track

Reply:
I would recommending having versions in the filename, at least at this stage of the project. When debugging and when people report problems we can at least have an idea what version is being run.

I think it's manageable for referencing as long as we name according to a fixed schema of `vX.Y.Z` or `vX.Y`.